readme.md

```diff
@@ -124,17 +124,23 @@ If you don't have root access, just install everything on your user's folder and
 
 This is an example on how to record the system update package count daily.  
 This will update the file `/var/tmp/updatecount.txt` for later access.  
-Root privilege is required for this operation.
+Root privilege is required for this operation. 
+Only /etc/crontab and the files in /etc/cron.d/ have a username field.
+ 
+##### So there you can do this:
 
-##### To edit root's crontab
 ```bash
-sudo crontab -e
+sudo vim /etc/crontab
```

Comment:
Thanks @ThomDietrich `/etc/cron.d/firemotd` might be the better option indeed. But a lot of users ad systems I now still use `sudo crontab -e` and put in the user field, as per your example. Got some complaints it isn't working in those cases.